src/plone/restapi/serializer/local_roles.py

```diff
@@ -17,10 +17,11 @@ def __init__(self, context, request):
         self.context = context
         self.request = request
 
-    def __call__(self):
+    def __call__(self, search=None):
+        self.request.form['search_term'] = search
         sharing_view = getMultiAdapter((self.context, self.request),
                                        name='sharing')
-        local_roles = sharing_view.existing_role_settings()
+        local_roles = sharing_view.role_settings()
```

Comment:
@jaroel this changes the structure of what we return on the sharing view, correct? If this is the case, this is a breaking change.